packages/styles/globals.import.styl

```diff
@@ -0,0 +1,136 @@
+html
+body
+  height 100%
+
+body
+  @extend .body-font
+  color $body-text
+
+h1
+h2
+h3
+h4
+h5
+  @extend .header-font
+  color $primary
+  font-weight 700
+
+h1
+  font-size 3em
+  font-weight 800
+
+h2
+  font-size 2em
+  color $primary
+
+h3
+  font-size 1.5em
+  color $secondary
+
+h4
+  color $body-text
+  font-weight 400
+  font-size 1.5em
+
+h5
+  font-size 1.25em
+  font-weight 400
+  color $d-gray
+
+a
+button
+.btn
+  transition()
+  &:hover
+    transition()
+
+a
+  color $primary-strong
+  &:hover
+  &:focus
+    color $primary
+
+textarea
+  display block
+  width 100%
+  height 34px
+  padding 6px 12px
+  border 1px solid #ccc
+  border-radius 4px
+  box-shadow inset 0 1px 1px rgba(0, 0, 0, .075)
+  background-color #fff
+  background-image none
+  font-size 14px
+  line-height 1.42857143
+  color #555
```

Comment:
Turns out this block isn't even necessary. I must have added it to Rana before we applied `form-control` class names to textareas. I'll remove it.